docker-compose.yml

```diff
@@ -8,9 +8,9 @@ web:
   environment:
     - PYTHONUNBUFFERED=1
     - PYTHONASYNCIODEBUG=1
-    - GENE_TO_REACTIONS_API
-    - GENE_TO_REACTIONS_PORT
-    - ID_MAPPER_API
+    - GENE_TO_REACTIONS_API=139.59.134.152
```

Comment:
I think it'd be nice to switch to Docker compose version 2.1 allowing the use of environment variables with default values. That allows me to have an `.env` file in my local directory with the real values that I might need to/want to use rather than having a dirty `docker-compose.yml` lying around or even commit a `docker-compose.yml` just to change the values. So I'd prefer to write them as follows: `GENE_TO_REACTIONS_API=${GENE_TO_REACTIONS_API:-139.59.134.152}`.